src/cloudai/workloads/ai_dynamo/aiperf.sh

```
@@ -0,0 +1,143 @@
+
+  log "Launching aiperf: ${run_cmd[*]} --model $model --url $full_url"
+
+  "${run_cmd[@]}" \
+    --model         "$model" \
+    --url           "$full_url" \
+    --endpoint-type chat \
+    --streaming \
+    --artifact-dir  "$artifact_dir" \
+    --no-server-metrics \
+    "${aiperf_profile_args[@]}" \
+    ${extra_args}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)